e2e/steps_file.js

```diff
@@ -153,6 +153,15 @@ module.exports = function () {
       this.dontSeeElement(caseListPage.locateCase(normalizeCaseId(caseId)));
     },
 
+    async seeEndStateForEvent(eventName, state) {
```

Comment:
this looks messy but I don't know a much better way of doing it 😄

Reply:
We could either go into the case list page and search for the case or we could check the history tab which is what I am doing here. I have slightly over engineered this as it will select the first instance of the event if it isn't already selected so that it can work if you want to check the most recent instance of a prior event.

Reply:
If you think it is overly complex I can remove the try block

Reply:
should be fine just wanted to double check 👍